diesel/src/query_dsl/save_changes_dsl.rs

```diff
@@ -79,30 +81,38 @@ pub trait SaveChangesDsl<Conn> {
     /// # include!("../doctest_setup.rs");
     /// # use schema::animals;
     /// #
-    /// # #[derive(AsChangeset, Debug, PartialEq, Identifiable, Queryable)]
-    /// # struct Animal {
-    /// #    id: i32,
-    /// #    species: String,
-    /// #    legs: i32,
-    /// #    name: Option<String>,
+    /// #[derive(Queryable, Debug, PartialEq)]
+    /// struct Animal {
+    ///    id: i32,
+    ///    species: String,
+    ///    legs: i32,
+    ///    name: Option<String>,
+    /// }
+    ///
+    /// #[derive(AsChangeset, Identifiable)]
+    /// #[table_name = "animals"]
+    /// struct AnimalForm<'a> {
+    ///     id: i32,
+    ///     name: &'a str,
+    /// }
+    ///
+    /// # fn main() {
+    /// #     run_test();
     /// # }
     /// #
-    /// # fn main() {
+    /// # fn run_test() -> QueryResult<()> {
     /// #     use animals::dsl::*;
     /// #     let connection = establish_connection();
-    /// let mut spider = animals.filter(species.eq("spider"))
-    ///     .first::<Animal>(&connection)
-    ///     .expect("Too scary to load");
-    ///
-    /// spider.species = String::from("solifuge");
-    /// spider.legs = 10;
-    ///
-    /// spider.save_changes::<Animal>(&connection).expect("Error saving changes");
-    ///
-    /// let changed_animal = animals.find(spider.id())
-    ///     .first::<Animal>(&connection);
-    ///
-    /// assert_eq!(Ok(Animal { id: 2, species: "solifuge".to_string(), legs: 10, name: None }), changed_animal);
+    /// let form = AnimalForm { id: 2, name: "Super scary" };
+    /// let changed_animal = form.save_changes(&connection)?;
+    /// let expected_animal = Animal {
+    ///     id: 2,
+    ///     species: String::from("spider"),
+    ///     legs: 8,
+    ///     name: Some(String::from("Super scary")),
```

Comment:
Really? "Super scary"? That's what you'd name your pet spider? I'd go for something like "Charlotte".

Reply:
Originally I was going with "might eat your fucking face if you take your eyes off it"

Reply:
Damn, what kind of spiders do you have in New Mexico…

Reply:
Never trust something with more than 2 eyes

Reply:
Feel free to change the name, I'm just being lazy